### ISSUE TITLE
Improve keyboard shortcuts in inputs

### ISSUE THREAD

**Issue by starius** [mikedilger/gossip#411]
I got used to remove last written word using Ctrl+Backspace. In Gossip this results in removing the whole message. Fortunately Ctrl+Z works, but please fix Ctrl+Backspace.

Ctrl+Left jumps to the beginning of input, should jup one word left.

**Comment by markwaters** [comment]
Are there keyboard shortcuts for scrolling the feed , etc ?

**Comment by Nate-Levin** [comment]
I cant seem to replicate this. Are you on windows or Linux? Possibly been fixed upstream with egui

**Comment by mister-monster** [comment]
> Are there keyboard shortcuts for scrolling the feed , etc ?

yes, but they appear to only work for me while the cursor is moving over the main pane. So, if i push down/up/pgdn/pgup it doesnt move except to the top of the next note, unless im wobbling my mouse around. weird.

Scrolling is also extremely slow and cumbersome with the mouse.

**Comment by mikedilger** [comment]
> Scrolling is also extremely slow and cumbersome with the mouse.

This is not my experience. Must be system dependent.  Please try tweaking the Settings>Ui settings:
* Try turning off Inertial Scrolling
* Try adjusting the Mouse acceleration
